metricbeat/module/vsphere/virtualmachine/_meta/fields.yml

```diff
@@ -4,6 +4,7 @@
     virtualmachine
   release: beta
   fields:
+    # ECS and potentially on module level?
     - name: host
```

Comment:
Yep, vm.host => host.hostname. ECS doesn't have anything for vm.name, though.

Making a note to look into this.

Reply:
Skipping.